src/HOHQMesh.jl

```diff
@@ -7,25 +7,19 @@ using HOHQMesh_jll: HOHQMesh_jll
 using Requires: @require
 
 function __init__()
-  # Enable features that depend on the availability of the Makie package
-  @require Makie="ee78f7c6-11fb-53f2-987a-cfe4a2b5a57a" begin
-    using .Makie
-    if isdefined(Makie, :to_textsize)
-      @warn "You seem to be using an older version of Makie (< v0.19). Some plotting functions may not work."
```

Comment:
The Makie compat setting requires v0.20 instead of v0.19 mentioned here. Is that correct?

Reply:
The Makie v0.20 is the one that deprecated a keyword argument that was removed in #50 . I can bump this back down to v0.19 to see what happens, but I expect some warning will get thrown.

Reply:
As expected, if I set the compat bound to v0.19 (or lower) the precompilation as well as when I run and visualize the interactive tool throw some deprecation warnings. Everything still works however.

Reply:
You could change the compat setting in Project.toml to `0.19, 0.20` - but I'm fine keeping it as it is right now in the PR